extensions/basic-ui/src/AppScreen.tsx

```diff
@@ -115,6 +115,7 @@ const AppScreen: React.FC<AppScreenProps> = ({
   const zIndexBase = zIndex * 4;
   const zIndexDim = zIndexBase;
   const zIndexPaper = zIndexBase + (theme === "cupertino" && hasAppBar ? 1 : 3);
+  const zIndexEdge = zIndexBase + 4;
   const zIndexAppBar = zIndexBase + 6;
```

Comment:
조금 더 명시적으로 요렇게 바꾸는건 어떻게 생각하세요??

```suggestion
  const zIndexBase = zIndex * 5;
  const zIndexDim = zIndexBase;
  const zIndexPaper = zIndexBase + (theme === "cupertino" && hasAppBar ? 1 : 3);
  const zIndexEdge = zIndexBase + 4;
  const zIndexAppBar = zIndexBase + 7;
```

Reply:
이전 화면의 zIndexEdge가 다음 화면의 zIndexDim과 겹칠거같아서요 ㅎㅎ HTML DOM 순서로 인해서 상하 관계가 정해지긴할텐데, 조금 더 명확하게 하면 좋을거같아요!

Reply:
오오.. 좋아욥. base가 0으로 잡혀서 겹치는 경우를 놓쳤었네요